session.go

```diff
@@ -152,6 +152,9 @@ func (sess *session) handleRequests(reqs <-chan *gossh.Request) {
 			var kv = struct{ Key, Value string }{}
 			gossh.Unmarshal(req.Payload, &kv)
 			sess.env = append(sess.env, fmt.Sprintf("%s=%s", kv.Key, kv.Value))
+			if req.WantReply {
+				req.Reply(true, nil)
+			}
 		case "pty-req":
```

Comment:
How should this handle multiple pty-req messages before the exec/shell? Right now I think this will overwrite the pty, but if that's what we want, we need to be more careful about closing sess.winch because that could be double-closed.